modules/deploy/Rakefile

```diff
@@ -52,8 +52,15 @@ task :setup_secrets do
     require 'securerandom'
     @secrets[:couchdb_password] = SecureRandom.hex
   end
+  if not ENV["ERLANG_COOKIE"].nil?
+    @secrets[:erlang_cookie] = ENV["ERLANG_COOKIE"]
+  else
+    puts "WARNING: ERLANG_COOKIE env variable is not set, setting it to a random string"
+    require 'securerandom'
```

Comment:
Now that this module is used in two places, please move the require to the top of the file.